CMakeLists.txt

```diff
@@ -177,7 +177,7 @@ if(BUILD_GUI)
   if(BUILD_GUI_TESTS)
     list(APPEND qt_components Test)
   endif()
-  find_package(Qt5 5.11.3 MODULE REQUIRED
+  find_package(Qt 5.11.3 MODULE REQUIRED
```

Comment:
It seems to me we're still hardcoding the major version - can't we change this during migration instead?

Reply:
> It seems to me we're still hardcoding the major version

I meant the variable names. The full minimum required version was not supposed to change.

> can't we change this during migration instead?

It is done in https://github.com/bitcoin/bitcoin/pull/30997. This PR split from it to make the https://github.com/bitcoin/bitcoin/pull/30997 easier to review.